app/pages/_app.tsx

```diff
@@ -1,7 +1,9 @@
 import '../styles/globals.css'
+import { getFirestore, connectFirestoreEmulator } from "firebase/firestore"
```

Comment:
We import these from `firebase/firestore` but we don't do anything with them! We can get rid of these.